tests/api/admin/controller/test_custom_lists.py

```diff
@@ -986,6 +986,26 @@ def test_share_locally_with_invalid_entries(
         assert response["failures"] == 2
         assert response["successes"] == 0
 
+    def test_share_locally_with_entry_with_missing_work(
```

Comment:
It would also be good to verify that the new warning is logged when the condition is met, but not logged when the condition is not met. We can use the pytest `capsys` fixture or similar to access this information.

Reply:
Done - I used caplog :)